modules/internal/LocaleModelHelpSetup.chpl

```diff
@@ -154,16 +154,12 @@ module LocaleModelHelpSetup {
     dst.maxTaskPar = chpl_task_getMaxPar();
   }
 
-  proc helpSetupLocaleNUMA(dst:borrowed LocaleModel, out local_name:string, out numSublocales, type NumaDomain) {
+  proc helpSetupLocaleNUMA(dst:borrowed LocaleModel, out local_name:string, const in numSublocales, type NumaDomain) {
```

Comment:
Any reason to make this `const in` over just `in` or blank? (I'm fine leaving it, just curious if there was a particular reason.)

Reply:
No, I think my head just went there since it had an explicit intent before and I was trying to update it to what I wanted it to be.  I didn't want `in` because I didn't want the local routine to think it could modify it, but blank would be appropriate.